openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js

```diff
@@ -72,7 +72,7 @@ describe("Entity Announcement", () => {
     cy.get('[data-testid="inActive-announcements"]').should('be.visible');
 
     // close announcement drawer
-    cy.get('.anticon > svg').should('be.visible').click();
+    cy.get('.anticon-close > svg > path').should('be.visible').click();
```

Comment:
Why we are dig in till `path` element?

Reply:
@chirag-madlani  As we have set onClick on SVG and it has a path as a child in it.  Either we can pass { multiple: true } or just go one step in and add an event listener there.

Reply:
```
cy.get('[data-testid="announcement-drawer"]').find('.anticon > svg').should('be.visible').click();
```

Reply:
@ShaileshParmar11  Updated!